mis_builder/tests/test_multi_company_aep.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+# © 2014-2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import datetime
+import time
+
+from odoo import fields
+import odoo.tests.common as common
+from odoo.tools.safe_eval import safe_eval
+
+from ..models.aep import AccountingExpressionProcessor as AEP
+from ..models.accounting_none import AccountingNone
+
+
+class TestMultiCompanyAEP(common.TransactionCase):
+
+    def setUp(self):
+        super(TestMultiCompanyAEP, self).setUp()
+        self.res_company = self.env['res.company']
+        self.account_model = self.env['account.account']
+        self.move_model = self.env['account.move']
+        self.journal_model = self.env['account.journal']
+        self.currency_model = self.env['res.currency']
+        self.curr_year = datetime.date.today().year
+        self.prev_year = self.curr_year - 1
+        # create company A and B
+        self.companyA = self.res_company.create({
+            'name': 'AEP Company A',
+            'currency_id': self.currency_model.search([('name', '=', 'USD')])})
```

Comment:
Search on currency_model would return the model and not just id so changing this here and below would make travis go green

Reply:
@karan-dreambits a lot more than that is needed to make travis go green.

I started reworking that test a bit to make it shorter and focus on multi company stuff (we don't need to retest everything that is already covered in test_aep.py)